extensions/map-layers/src/ui/widget/MapUrlDialog.tsx

```diff
@@ -76,8 +76,7 @@ export function MapUrlDialog(props: MapUrlDialogProps) {
   const [nameInputPlaceHolder] = React.useState(MapLayersUiItemsProvider.i18n.translate("mapLayers:CustomAttach.NameInputPlaceHolder"));
   const [urlLabel] = React.useState(MapLayersUiItemsProvider.i18n.translate("mapLayers:CustomAttach.URL"));
   const [urlInputPlaceHolder] = React.useState(MapLayersUiItemsProvider.i18n.translate("mapLayers:CustomAttach.UrlInputPlaceHolder"));
-  // SWB
-  const [projectSettingsLabel] = React.useState(MapLayersUiItemsProvider.i18n.translate("mapLayers:CustomAttach.StoreOnProjectSettings"));
+  const [iTwinSettingsLabel] = React.useState(MapLayersUiItemsProvider.i18n.translate("mapLayers:CustomAttach.StoreOnITwinSettings"));
```

Comment:
Won't this change break i18n?

Reply:
Yes, you'll need to update the string in the public/locales/*.json file as well.

Reply:
Updated the localization